dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/KeyJsonValueController.java

```diff
@@ -100,6 +106,31 @@
         return keys;
     }
 
+    @GetMapping( value = "/{namespace}", params = "fields", produces = APPLICATION_JSON_VALUE )
```

Comment:
BTW: This can use the same path as the already existing GET mapping since we only map to this method when the `fields` parameter is present. This way the API can stay nice as if the same endpoint would support being used with or without `fields` parameter.